database/schema/12-feegrant.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE fee_grant_allowance
+(
+    id                 SERIAL      NOT NULL PRIMARY KEY,
+    grantee            TEXT        NOT NULL REFERENCES account (address),
+    granter            TEXT        NOT NULL REFERENCES account (address),
```

Comment:
I think we should rename those so that the GraphQL relationships will have better names: 
```sql
    grantee_address            TEXT        NOT NULL REFERENCES account (address),
    granter_address            TEXT        NOT NULL REFERENCES account (address),
```